### ISSUE TITLE
disconnected: client is to old

### ISSUE THREAD

**Issue by Thystro** [akezeke/spotyxbmc#30]
Message after entering username and password: 

Disconnected:
Client is to old, library will need to be updated.
Connect?

clicking on yes doesn't help

**Comment by akezeke** [comment]
Yes, it is too old. Use spotyxbmc2 instead with libspotify 10! Spotyxbmc
(1) is not supported anymore.

Kind regards
David
Den 2 dec 2011 17:06 skrev "Thystro" <
reply@reply.github.com

**Comment by Thystro** [comment]
Just noticed it indeed! Browsing the XBMC forum as we speak. Maybe you should inform spotify to change the URL they refer to when you click on "spotyxbmc"

http://www.spotify.com/nl/about/spotify-on-the-web/apps-media-player-addons/
